app/models/miq_report/seeding.rb

```diff
@@ -84,12 +86,34 @@ def seed_record(path, report)
       end
     end
 
+    def template_type(path)
+      path.include?(RELATIVE_REPORT_DIR.to_s) ? "report" : "compare"
+    end
+
     def seed_files
+      seed_core_files + seed_plugin_files
+    end
+
+    def seed_core_files
       Dir.glob(REPORT_DIR.join("**/*.yaml")).sort + Dir.glob(COMPARE_DIR.join("**/*.yaml")).sort
     end
 
+    def seed_plugin_files
+      Vmdb::Plugins.flat_map do |plugin|
+        Dir.glob(plugin.root.join("#{RELATIVE_REPORT_DIR}/**/*.yaml")).sort + Dir.glob(plugin.root.join("#{RELATIVE_COMPARE_DIR}/**/*.yaml")).sort
+      end
+    end
+
     def seed_filename(path)
-      path.remove("#{REPORT_DIR}/").remove("#{COMPARE_DIR}/")
+      send("seed_#{template_type(path)}_filename", path)
+    end
+
+    def seed_report_filename(path)
+      path.split("#{RELATIVE_REPORT_DIR}/").last
```

Comment:
I used `.remove` previously because I thought it was simpler than `.split().last`.  Thoughts?

Reply:
That worked when what was only `Rails.root` before the reports dir, now I'd have to remove all of the provider repos as well